core/modules/user/user.pages.inc

```diff
@@ -48,9 +48,8 @@ function user_pass() {
     $form['name']['#type'] = 'value';
     $form['name']['#value'] = $user->mail;
     $form['mail'] = array(
-      '#prefix' => '<p>',
+      '#type' => 'help',
       '#markup' =>  t('Password reset instructions will be mailed to %email. You must log out to use the password reset link in the e-mail.', array('%email' => $user->mail)),
-      '#suffix' => '</p>',
```

Comment:
Not sure about this usage either because it's in the middle of the form. All other places are help as displayed at the top of the page. If help elements were styled, they might be given a visual wrapper, icon, and margins, which might not work well in the middle of a page.

Reply:
Sure, done.